lumen/tests/hello_world.rs

```diff
@@ -13,6 +13,8 @@ mod hello_world {
             .arg("_build")
             .arg("-o")
             .arg("hello_world")
+            // Turn off optimizations as work-around for debug info bug in EIR
```

Comment:
I'd prefix these with `HACK` or `TODO` just to make sure we can find these notes easily later on